core-test/src/main/resources/network/sfc/port_pair_group.json

```diff
@@ -1,13 +1,14 @@
 {
     "port_pair_group": {
         "id" : "abcdef",
+        "group_id": 42,
```

Comment:
there is no group id for port pair groups

https://docs.openstack.org/networking-sfc/latest/contributor/api.html

Reply:
our openstack deployment returned it. I thought it was a gap in the docs.